commit a5cc1f334aca
Author: Jeff Gilbert <jgilbert@mozilla.com>
Date:   Sat Mar 3 00:50:10 2018 +0200

    Bug 1442504 - Disable disjoint timer queries. r=milan, a=RyanVM
    
    MozReview-Commit-ID: IurPcGHzAoQ
    
    --HG--
    extra : source : bef3db82d7ca31a188f902e317713c88001938ed
---
 dom/canvas/WebGLContextExtensions.cpp         | 4 ++--
 dom/canvas/test/webgl-mochitest/mochitest.ini | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git dom/canvas/WebGLContextExtensions.cpp dom/canvas/WebGLContextExtensions.cpp
index 120ca85a01a2..04cbbbe6b31d 100644
--- dom/canvas/WebGLContextExtensions.cpp
+++ dom/canvas/WebGLContextExtensions.cpp
@@ -93,6 +93,8 @@ bool WebGLContext::IsExtensionSupported(dom::CallerType callerType,
 
     if (allowPrivilegedExts) {
         switch (ext) {
+        case WebGLExtensionID::EXT_disjoint_timer_query:
+            return WebGLExtensionDisjointTimerQuery::IsSupported(this);
         case WebGLExtensionID::MOZ_debug:
             return true;
         case WebGLExtensionID::WEBGL_debug_renderer_info:
@@ -118,8 +120,6 @@ WebGLContext::IsExtensionSupported(WebGLExtensionID ext) const
     switch (ext) {
     // In alphabetical order
     // EXT_
-    case WebGLExtensionID::EXT_disjoint_timer_query:
-        return WebGLExtensionDisjointTimerQuery::IsSupported(this);
     case WebGLExtensionID::EXT_texture_filter_anisotropic:
         return gl->IsExtensionSupported(gl::GLContext::EXT_texture_filter_anisotropic);
 
diff --git dom/canvas/test/webgl-mochitest/mochitest.ini dom/canvas/test/webgl-mochitest/mochitest.ini
index d36e66b7cdbd..08e2dd335dce 100644
--- dom/canvas/test/webgl-mochitest/mochitest.ini
+++ dom/canvas/test/webgl-mochitest/mochitest.ini
@@ -19,7 +19,7 @@ fail-if = (os == 'android')
 [ensure-exts/test_EXT_color_buffer_half_float.html]
 fail-if = (os == 'android')
 [ensure-exts/test_EXT_disjoint_timer_query.html]
-fail-if = (os == 'android') || (os == 'mac') || (os == 'win' && os_version == '5.1')
+fail-if = 1
 [ensure-exts/test_EXT_frag_depth.html]
 fail-if = (os == 'android')
 [ensure-exts/test_EXT_sRGB.html]
